_data/projects/github-practice-for-all.yml

```diff
@@ -0,0 +1,11 @@
+name: Github-practice-for-all
+desc: Here you are welcome to test how team workflow might look and practice git pull, git push and more...
+site: https://github.com/RomchyFCC/GitHub-Practice-For-All
+tags:
+- github
+- practice
+- pull
+- push
+up-for-grabs:
+  name: add-name-to-file

```

Comment:
@RomchyFCC Please change it to "up-for-grabs" as your label name

Reply:
Done, ty